src/features/game/Game.tsx

```diff
@@ -49,6 +50,7 @@ const SHOW_MODAL: Record<StateValues, boolean> = {
   refreshing: true,
   expanded: false,
   expanding: false,
+  needSync: true,
```

Comment:
I'm wondering if we can tie this closer to the actual domain of the game and whether `needSync` is just a technical term for the state?

Could we call this the `hoarding` state?